lua/core/lualine/styles.lua

```diff
@@ -150,13 +150,24 @@ styles.lvim = {
           end
           local buf_ft = vim.bo.filetype
           local buf_client_names = {}
+
+          -- add client
           local active_client = require("lsp.utils").get_active_client_by_ft(buf_ft)
           for _, client in pairs(buf_clients) do
             if client.name ~= "null-ls" then
               table.insert(buf_client_names, client.name)
             end
           end
           vim.list_extend(buf_client_names, active_client or {})
+
+          -- add formatter
+          local active_formatters = require("lsp.null-ls.formatters").list_supported_names(buf_ft)
```

Comment:
https://github.com/luarocks/lua-style-guide#modules

Reply:
Thanks always.
I do not mean to make you over again with a same thing :(
But could you check it again and tell me more info pls?
It is under ```components.lua``` now